heuristic_baseline.py

```diff
@@ -0,0 +1,44 @@
+from dataloader import *
+from model import heuristic
+import numpy as np
+import torch
+from matplotlib import pyplot as plt
+
+
+class heuristic_run:
+    """
+    The class to run the heuristic model. No training needed.
+    Parameters can be tuned in model.heuristic
+    """
```

Comment:
This seems like a well written class, but it seems overly specific to the heuristic model. My instinct is this can be generalized to not just the heuristic model, but any other model. I.E., it outputs metrics like accuracy and graphs. It really feels like there's more.